py_neuromodulation/nm_stream.py

```diff
@@ -134,23 +140,38 @@ def _run(
                 data_batch.astype(np.float64)
             )
             if is_stream_lsl:
-                feature_dict["time"] = time_[-1]
                 if self.verbose:
```

Comment:
Not sure about this calculation of the batch time. I think that what should be being saved is either the first time of the batch or the last, not the length of the data. Also I think it's not correct that the first batch does not show the seconds of data processed. I attempted to fix it in my commit.